ds701_book/11-Dimensionality-Reduction-SVD-II.qmd

```diff
@@ -1,857 +1,759 @@
+Output: $\mathbf{y}_1,\dots, \mathbf{y}_m$  with  $\mathbf{y}_i \in \mathbb{R}^d \: \: \forall \: i \in \{1, \dots, n\}$. 
 
-:::: {.fragment}
-__Find the $k$-dimensional hyperplane that is "closest" to the points.__
-::::
+The goal is to compute the new data points $\mathbf{y}_i$ such that <font color="red"> $d << n$ </font> while still preserving the most information contained in the data points $\mathbf{x}_i$.
 
-:::: {.fragment}
-More precisely:
+$$
+X_0 = 
+\begin{bmatrix} 
+x_{11} & x_{12} & \dots & x_{1n} \\
+x_{21} & x_{22} & \dots & x_{2n} \\
+\vdots & \vdots & \ddots & \vdots \\
+x_{m1} & x_{m2} & \dots & x_{mn} 
+\end{bmatrix} \:
+\xrightarrow[\text{PCA}]{} \:
+Y = \begin{bmatrix} 
+y_{11} & y_{12} & \dots & y_{1d} \\
+y_{21} & y_{22} & \dots & y_{2d} \\
+\vdots & \vdots & \ddots & \vdots \\
+y_{m1} & y_{m2} & \dots & y_{md} 
+\end{bmatrix}
+$$
 
-Given n points in $\mathbb{R}^n$, find the hyperplane (affine space) of dimension $k$ with the property that the squared distance of the points to their orthogonal projection onto the hyperplane is minimized.
-::::
+## PCA Application: Genes Mirror Geography
 
+We consider a dataset from @novembre2008genes. The authors collected DNA data called SNPs (single nucleotide polymorphism) from 3000 individuals in Europe. 
 
-::: {.content-visible when-profile="slides"}
-## Dimensionality reduction continued
-:::
+SNPs describe the changes in DNA from a common base pair (A,T,C,G). A value of 0 means no changes to the base pair, a value of 1 means 1 change to the base pair, and a value of 2 means both base pairs change.
 
-![](figs/pca_figure1.jpeg){fig-align="center" width="60%"}
+The data for each individual consisted of approximately 500k SNPs.
 
-[Source](https://liorpachter.wordpress.com/2014/05/26/what-is-principal-component-analysis/)
+---
 
-::: {.content-visible when-profile="web"}
-This sounds like an appealing criterion.   
+The authors performed PCA and plotted 1387 of the individuals in the reduced dimensions.
 
-But it also turns out to have a strong statistical guarantee.
-:::
+![Image Credit @novembre2008genes](figs/PCA_genes_Europe.png)
+
+---
 
-::: {.content-visible when-profile="slides"}
-## Dimensionality reduction continued
+Key observations:
 
-__Find the $k$-dimensional hyperplane that is "closest" to the points.__
-:::
+:::: {.incremental}
+- the first principal components of the data almost reproduce the map of Europe
+- SNPs are similar geographically
+- DNA of an individual reveals their birthplace within a few hundred kilometers
+::::
 
 :::: {.fragment}
-In fact, this criterion is a transformation that captures the maximum __variance__ in the data.
+Would a similar study in the USA be effective?
 ::::
 
-:::: {.fragment}
-That is, the resulting $k$-dimensional dataset is the one with maximum variance.
+## PCA Steps
```

Comment:
So the intent here is to now formally define the PCA steps? Maybe a little more setup saying that is what we are going to do. However, then we proceed with an example, right?

Reply:
Yes. I think the underlying tension is that I planned to explain a lot of this in words so the text is more slide like as opposed to more of a book.

Reply:
I added some clarifying remarks that this was an outline of what is done, but that a more formal procedure to do this will be provided.